### PR TITLE
:bear: :sparkles: added NewHookerFromCollection

### PR DESCRIPTION
**Pull request by brunetto**
New constructor to be able to create hook from existing connection

### CODE REVIEW

**Review by weekface — COMMENTED** [review]
This looks good to me, but can you add this to [usage](https://github.com/weekface/mgorus#usage) section?

**Comment by brunetto** [comment]
Done! 😄

File: mgorus.go [hunk 1]
```diff
@@ -55,9 +55,9 @@ func NewHookerWithAuthDb(mgoUrl, authdb, db, collection, user, pass string) (*ho
 
 func (h *hooker) Fire(entry *logrus.Entry) error {
 	data := make(logrus.Fields)
-	data["Level"] = entry.Level
-	data["Time"] = entry.Time
-	data["Message"] = entry.Message
+	data["level"] = entry.Level.String()
```
Comment (weekface) [inline mgorus.go]: Why changed to lower case? This may be a break change.

Comment (brunetto) [reply mgorus.go]: you're right, reverted to uppercase

**Comment by weekface** [comment]
Thank you.

**Pull request merged**